src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/Internal/HostTests.cs

```diff
@@ -1382,9 +1445,13 @@ public AsyncThrowingService(Task executeDelayTask)
 
             protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             {
-                await _executeDelayTask;
+                while (true)
```

Comment:
I don't understand what this `while (true)` is supposed to do.

Reply:
I was initially planning on adding more test cases that would more closely represent how background services are actually written. More often than not, they sit in while loops, and so some work periodically. I'll remove this as I didn't end up writing those additional test cases.